cli/cmd/cluster_kubeconfig.go

```diff
@@ -50,6 +50,7 @@ replicated cluster kubeconfig --id CLUSTER_ID`,
 
 	cmd.Flags().StringVar(&r.args.kubeconfigClusterName, "name", "", "name of the cluster to download credentials for (when id is not provided)")
 	cmd.RegisterFlagCompletionFunc("name", r.completeClusterNames)
+	cmd.Flag("name").Deprecated = "use ID_OR_NAME arguments instead"
```

Comment:
We should add a similar deprecation warning to `cmd.Flag("id")` for commands that specify that.